custom/theme.scss

```diff
@@ -1,6 +1,6 @@
-$light-background: #e9e5fb;
```

Comment:
Consider using consistent naming conventions for color variables to maintain a unified style. 

Suggested change:
```scss
$light-background: #F8EFF4;
``` 

For reference, see [Sass Guidelines](https://sass-lang.com/guide).